 07 may 2021: patch is possibly no longer needed on www-client/firefox-88.0.1 at least, as per: https://bugzilla.mozilla.org/show_bug.cgi?id=1558101#c5

seems caused by https://bugzilla.mozilla.org/show_bug.cgi?id=1558101#c4

diff -r e704e3565db9 dom/base/DOMQuad.h
--- a/dom/base/DOMQuad.h	Thu Jul 11 00:49:58 2019 +0300
+++ b/dom/base/DOMQuad.h	Thu Jul 11 05:28:08 2019 +0200
@@ -16,6 +16,7 @@
 #include "mozilla/dom/BindingDeclarations.h"
 #include "mozilla/ErrorResult.h"
 #include "Units.h"
+#include "DOMRect.h"
 
 class nsIGlobalObject;
 
